Gordon360/Properties/launchSettings.json

```diff
@@ -3,12 +3,12 @@
     "Development": {
       "commandName": "Project",
       "launchBrowser": true,
-      "launchUrl": "https://localhost:5100/swagger",
+      "launchUrl": "https://localhost:51657/swagger",
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
       },
       "sqlDebugging": true,
-      "applicationUrl": "https://localhost:5100;http://172.26.160.1:51626"
+      "applicationUrl": "https://localhost:51657;http://localhost:51658"
```

Comment:
Same here @RossClark01

Reply:
These changes are fine, in the sense that the settings here are arbitrary and should be changed by every developer.